Signal/src/call/CallService.swift

```diff
@@ -470,9 +470,20 @@ protocol CallServiceObserver: class {
 
         let newCall = SignalCall.incomingCall(localId: UUID(), remotePhoneNumber: thread.contactIdentifier(), signalingId: callId)
 
+        guard self.call == nil else {
```

Comment:
Why this reordering? I think we should prefer *not* to send anything (even a busy signal) in this scenario.